sqlmesh/core/engine_adapter/shared.py

```diff
@@ -158,6 +158,45 @@ def supports_column_comment_commands(self) -> bool:
         )
 
 
+class CommentSupportMaterializedView(Enum):
```

Comment:
Are there other engines that should override this today? It looks a bit overkill to only handle BigQuery's case, maybe we can simplify and then revisit later if there's a need to expand our functionality?

Reply:
There are not other engines, and because BQ overrides `_create_column_comments` we can turn this off for materialized views there.

Just wasn't sure whether it would be better to align with the existing comment support approach (flags, this PR) vs. doing it that way.

Reply:
I'd probably deal with this locally in BigQuery vs introducing a new flag / enum class for now. If needed, we can always revisit and refactor to use this approach.